gapis/api/vulkan/draw_call_mesh.go

```diff
@@ -293,7 +293,7 @@ func getVerticesData(ctx context.Context, s *api.State, thread uint64,
 	fullSize := uint64(vertexCount-1)*stride + perVertexSize
 
 	offset := uint64(attribute.Offset) + (uint64(firstVertex) * stride)
-	if offset > vertexSlice.count || offset+fullSize > vertexSlice.count {
+	if offset >= vertexSlice.count || offset+fullSize > vertexSlice.count {
```

Comment:
@AWoloszyn, @Qining:
Given that `offset` and `fullSize` are both unsigned, do you need the LHS of the `||`.
Secondly, don't you have exactly the same off-by-one bug on the RHS?